packages/browser/src/util/Fetcher.ts

```diff
@@ -27,22 +27,23 @@
 /**
  * A wrapper method to wrap the standard w3 fetch library
  */
-import URL from "url-parse";
 import _fetch from "cross-fetch";
 
 export interface IFetcher {
-  fetch(url: RequestInfo | URL, init?: RequestInit): Promise<Response>;
+  fetch(url: RequestInfo | string, init?: RequestInit): Promise<Response>;
```

Comment:
```suggestion
  fetch(url: RequestInfo, init?: RequestInit): Promise<Response>;
```
`RequestInfo` is already a `Request | string`